osm2gtfs/creators/trips_creator.py

```diff
@@ -13,32 +19,302 @@ def __repr__(self):
         return rep
 
     def add_trips_to_feed(self, feed, data):
-        raise NotImplementedError("Should have implemented this")
+        """
+        This function generates and adds trips to the GTFS feed.
+
+        It is the place where geographic information and schedule is
+        getting joined to produce a routable GTFS.
+        """
```

Comment:
Maybe add a link to the expected schedule format here?

Reply:
Yes, I added the link to the wiki.